src/averageAges.js

```diff
@@ -1,64 +1,63 @@
 'use strict';
 
-/**
- * Implement calculateMenAverageAge function
- *
- * Function returns average age of men in array. If `century` is specified then
- * function calculates average age only for men who died in this century
- *
- * To calculate century:
- * Divide year of person's death by 100: Math.ceil(person.died / 100)
- *
- * @param {object[]} people
- * @param {number} century - optional
- *
- * @return {number}
- */
+const FEMALE = 'f';
+const MALE = 'm';
+
+function sumAges(cadaver) {
+  return cadaver.reduce((accum, person) =>
+    (accum + (person['died'] - person['born'])), 0) / cadaver.length;
+};
+
+function getAgeDiff(cadaverChildren, cadaver) {
+  return ((cadaverChildren.reduce((accum, child) =>
+    accum + child['born'] - cadaver.find(mom => child['mother']
+      === mom['name'])['born'], 0)) / cadaverChildren.length).toFixed(2);
+};
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const allMen = people.filter(person => person['sex'] === MALE);
```

Comment:
```suggestion
  const allMen = people.filter(({ sex })=> sex === MALE);
```

Reply:
It's better to do like this(`person.sex` not `person['sex']`), when it's possible
But even better would be to use destructuring
Please, fix this in all places